src/dialog/dialog.component.ts

```diff
@@ -133,43 +133,14 @@ export class Dialog implements OnInit, AfterViewInit, OnDestroy {
 		if (getFocusElementList(this.dialog.nativeElement).length > 0) {
 			dialogElement.focus();
 		}
-		const parentEl: HTMLElement = this.dialogConfig.parentRef.nativeElement;
-		let node = parentEl;
-		let observables = [];
-
-		// if the element has an overflow set as part of
-		// its computed style it can scroll
-		const isScrollableElement = (element: HTMLElement) => {
-			const computedStyle = getComputedStyle(element);
-			return (
-				computedStyle.overflow === "auto" ||
-				computedStyle.overflow === "scroll" ||
-				computedStyle["overflow-y"] === "auto" ||
-				computedStyle["overflow-y"] === "scroll" ||
-				computedStyle["overflow-x"] === "auto" ||
-				computedStyle["overflow-x"] === "scroll"
-			);
-		};
-
-		const isVisibleInContainer = (element, container) => {
-			const elementRect = element.getBoundingClientRect();
-			const containerRect = container.getBoundingClientRect();
-			return elementRect.bottom <= containerRect.bottom && elementRect.top >= containerRect.top;
-		};
+		const parentElement: HTMLElement = this.dialogConfig.parentRef.nativeElement;
```

Comment:
Does it need to be defined as an `HTMLElement`?

Reply:
Let me check, but maybe 🤔

Reply:
nope! it's already typed as `HTMLElement` from `.nativeElement`